cmd/broker/main.go

```diff
@@ -31,12 +36,23 @@ func main() {
 		logger.Fatal("main", err)
 	}
 
-	cli, err := client.NewEnvClient()
-	if err != nil {
-		logger.Fatal("docker-client", err)
-	}
+	var manager containermanager.ContainerManager
 
-	manager := docker.NewDockerContainerManager(logger, cli)
+	if cli, err := client.NewEnvClient(); err != nil {
```

Comment:
let's make it configurable in the config file to choose the underlying engine to run the broker.

We can have smoething like `"engine": "docker"` or `"engine": "kubernetes"` in the config file and decide on what to use based on what the property is set to. defaulting to `docker`.

Reply:
done in #52!